### PR TITLE
[MRG] Cleanup readme

### PR DESCRIPTION
**Pull request by cbrnr**
Synced with the description from the homepage

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,80 +24,86 @@
...
-Get more information
-^^^^^^^^^^^^^^^^^^^^
 
-If you're unfamiliar with MNE, you can visit the
-`MNE homepage <http://mne-tools.github.io/>`_ for full user documentation.
+Documentation
+^^^^^^^^^^^^^
+
+Documentation for MNE-Python is available at the
+`MNE homepage <http://mne-tools.github.io/>`_.
```
Comment (agramfort) [inline README.rst]: please use:

http://martinos.org/mne/stable/index.html

as home address

File: README.rst [hunk 2]
```diff
@@ -106,7 +112,7 @@ MNE-Python is **BSD-licenced** (3 clause):
     This software is OSI Certified Open Source Software.
     OSI Certified is a certification mark of the Open Source Initiative.
 
-    Copyright (c) 2011, authors of MNE-Python
+    Copyright (c) 2011, authors of MNE-Python.
```
Comment (agramfort) [inline README.rst]: Copyright (c) 2011-2017

File: README.rst [hunk 3]
```diff
@@ -24,80 +24,86 @@
...
-MNE-Python, you can do:
+Installing MNE-Python
+^^^^^^^^^^^^^^^^^^^^
+
+To install the latest stable version of MNE-Python, you can use `pip <https://pip.pypa.io/en/stable/>`_ in a terminal:
 
 .. code-block:: bash
 
-    $ pip install mne
+    pip install mne
```
Comment (agramfort) [inline README.rst]: I would recommend:

pip install -U mne

File: README.rst (hunk 1 above)
Comment (larsoner) [reply README.rst]: I'd prefer `https://martinos.org/mne` as it's shorter

**Comment by agramfort** [comment]
+1

**Comment by cbrnr** [comment]
Done. I've replaced all github.io URLs and switched to https. Let's double-check all links.

**Comment by cbrnr** [comment]
Yeah, looks good, all links are working.

**Comment by larsoner** [comment]
Thanks @cbrnr

**Pull request merged**